BangazonAPI/Models/EmployeeTraining.cs

```diff
@@ -0,0 +1,9 @@
+﻿namespace BangazonAPI.Models
+{
+    public class EmployeeTraining
```

Comment:
You don't always necessarily need models for join tables-- it really depends on the situation. It's fine to merge them in now but I just wanted to put it on your radar so that you can go back and delete any you don't use at the end of the sprint

Reply:
Alright thanks, we will discuss it and keep it in mind during the sprint